src/server/scripts/Commands/cs_learn.cpp

```diff
@@ -105,6 +106,70 @@ class learn_commandscript : public CommandScript, public PlayerCommand
         return true;
     }
 
+    static bool HandleLearnAllMyLvlCommand(ChatHandler* handler, char const* /*args*/)
```

Comment:
Would be better to move the logic of this function in a separate file in order to let people reuse it for other purposes (custom scripts etc).

In this way, modders could create scripts that automatically teach you spells when you level up for instance.

Reply:
I don't agree with handling commands in other files than the designated ones

Reply:
I said "the logic", which means creating a new function elsewhere and using that one in `HandleLearnAllMyLvlCommand`

that function can be reused then in other scripts/modules/logics

Reply:
ah, I oversaw that :D